corehq/apps/userreports/tests/utils.py

```diff
@@ -146,3 +152,40 @@ def get_simple_xform():
         'VT': 'VT',
     })
     return xform.tostring()
+
+
+def load_data_from_db(table_name):
```

Comment:
this is only used in one place and it only checks the length of the results. Seems like a lot of unnecessary code.

Reply:
Addressed in other PR